packages/front/src/routes/auth/activate/[token]/components/StructureStep/StructureStep.controller.ts

```diff
@@ -0,0 +1,63 @@
+import { AgentJobTypeEnum } from "@api-subventions-asso/dto";
+import Dispatch from "$lib/core/Dispatch";
+import Store from "$lib/core/Store";
+import { isPhoneNumber } from "$lib/helpers/stringHelper";
+
+type Option = {
+    value: AgentJobTypeEnum;
+    label: string;
+};
+
+export default class StructureStepController {
+    private readonly dispatch: (_: string) => void;
+    public readonly options: Option[];
+    public readonly errors: Store<{ [key: string]: string | undefined }>;
+    private static errorMandatory = "Ce champ est obligatoire";
+    private readonly validators: Record<string, (value: any) => string | undefined>;
+    private readonly dirty: { [key: string]: boolean };
+
+    constructor() {
+        this.dispatch = Dispatch.getDispatcher();
+        this.options = [
```

Comment:
Dans un futur il y aura des étapes 3 avec d'autres options (i.e: périmètre). Il faut peut-être dès à présent créer des sous tableau ou renommer `this.options` par `this.posteOptions` ?

Reply:
renommé en `jobTypeOptions`